Formula/aircrack-ng.rb

```diff
@@ -18,25 +17,26 @@ class AircrackNg < Formula
   depends_on "pkg-config" => :build
   depends_on "sqlite"
   depends_on "openssl"
+  depends_on "make"
+  depends_on "pcre"
 
   # Remove root requirement from OUI update script. See:
   # https://github.com/Homebrew/homebrew/pull/12755
   patch :DATA
```

Comment:
The patch has to do with aircrack-ng usually being root only. I think its just a home-brew thing and probably is on purpose that it is root only otherwise.